component/main.jsonnet

```diff
@@ -4,10 +4,14 @@ local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 
 local params = inv.parameters.openshift4_ingress;
+local ingressControllers =
```

Comment:
The Commodore library has a helper for this `com.getValueOrDefault()`.

Reply:
Unfortunately `getValueOrDefault` won't work in this case, because the key is there, it's just `null`. So we'd need to use `std.prune()`:

```
local ingressControllers = std.objectFields(com.getValueOrDefault(std.prune(params), 'ingressControllers', {}));
```

Which IMHO is not very readable/understandable... WDYT?

Reply:
I was not requesting a changes as I am fine with the way how it is now. I just wanted to hint this function. Yes, the null thing is a bummer. Using `std.prune` is a double edged sword. It already cut me. Some resources do need those empty values.